src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -120,6 +120,8 @@ function TransactionSubmittedContent({
       .catch(() => setSuccess(false))
   }, [connector, logoURL, token])
 
+  const explorerText = `View on ${chainId === ChainId.MAINNET ? 'Etherscan' : 'Block Explorer'}`
```

Comment:
This will not work.

```suggestion
  const explorerText = chainId === ChainId.MAINNET ? t`View on  Etherscan` : t`View on Block Explorer`
```

Reply:
Went with your suggestion but would it have worked if it wasn't an enriched string? ie: 
```
const explorerText = chainId === ChainId.MAINNET ? "View on  Etherscan" : "View on Block Explorer"
...
<Trans>{explorerText}</Trans>

Reply:
I don't know. I'd need to extract the messages to test it out.